_guide/_pages/security/cloud-services.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Cloud Services 
+sidenav: security
+sticky_sidenav: true
+---
+A common practice is storing files in the cloud; places like [Amazon Simple Storage Service](https://docs.aws.amazon.com/s3/index.html)
+(Amazon S3), [Google Cloud](https://cloud.google.com/storage/docs/introduction), or 
+[Azure Storage](https://docs.microsoft.com/en-us/rest/api/storageservices/delegate-access-with-shared-access-signature).
+These services give developer an easy way to store data in the cloud, and download it when needed. But this
+introduces security considerations.
+
+When thinking about how to upload or download data in your application, there are always tradeoffs to think about -- often
+processes that are easier to use are less secure; likewise a workflow that is more complex is often more secure. 
+
+These tradeoffs get more significant depending on the FISMA level of your system.
+
+### Presigned URLs 
+A common method of allowing users to transfer data without credentials is to use [presigned urls](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html).
+(Azure refers to these as [shared access signatures](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-storage-sharedaccesssignature-permissions), but they are a similar concept.)
+
+There are some differences between service providers; the below details are specific to S3 as that has
+been our most common use case and is supported by [cloud.gov](https://cloud.gov/docs/services/s3/).
+
+#### Some Things to know about S3 presigned URLs
+* They can be reused until they expire
+  * Default expiration time is 15 minutes
+* They can be used by *anyone*
+* There is no default file size limit on uploads
+* Uploads use PUT by default
+
+A presigned URL is an *easily-shareable* URL that is generated with an authenticated user's security credentials. They 
+are created with specific actions attached to them, as well as an expiration date and time; the URL will remain valid
+until that expiration moment.
+
+A shareable URL that bypasses security authentication is very convenient to use! However, the tradeoff is that anyone
+with access to that URL can use it. If a user for some reason decided to post a presigned upload URL to the internet,
+anyone could use that URL to upload data to your bucket until the expiration time was passed. 
+
+This introduces a "user error" or "insider threat" security vulnerability to your system. Even if your users are good actors,
+some attack vectors could be:
```

Comment:
(This is a nit so feel free to ignore.) "Vulnerability" feels maybe too strong here. If used properly & intentionally presigned URLs are features, not vulnerabilities ... But they create pathways that may make the consequences of certain user errors worse? Like I said, a nit!

Reply:
i think vulnerability is the right word here because it is exactly that; the URL is a feature, but it creates a vulnerable opening in the application. it might be a fine tradeoff, but i do think it's good to think about it in these terms!

Reply:
Fair!